rocAL/include/readers/image/caffe_lmdb_record_reader.h

```diff
@@ -106,8 +106,6 @@ class CaffeLMDBRecordReader : public Reader {
     //!< _file_count_all_shards total_number of files in to figure out the max_batch_size (usually needed for distributed training).
     size_t _file_count_all_shards;
     void incremenet_file_id() { _file_id++; }
-    void replicate_last_image_to_fill_last_shard();
-    void replicate_last_batch_to_pad_partial_shard();
```

Comment:
Please ensure we added the equivalent api's to do the similar operations as deleted api's